test/filecoin.test.js

```diff
@@ -202,18 +202,20 @@ test('w3filecoin integration flow', async t => {
   console.log(`put deal on deal tracker for aggregate ${aggregate}`)
   await putDealToDealTracker(aggregate.toString(), dealId)
 
-  // Trigger cron to update and issue receipts based on deals
-  const callDealerCronRes = await fetch(`https://staging.dealer.web3.storage/cron`)
-  t.true(callDealerCronRes.ok)
-
   // Await for `aggregate/accept` receipt
   const aggregateAcceptReceiptCid = receiptAggregateOfferRes.ok?.fx.join?.link()
   if (!aggregateAcceptReceiptCid) {
     throw new Error('aggregate/offer receipt has no effect for aggregate/accept')
   }
   console.log(`wait for aggregate/accept receipt ${aggregateAcceptReceiptCid.toString()} ...`)
   await waitForStoreOperationOkResult(
-    () => receiptStoreFilecoin.get(aggregateAcceptReceiptCid),
+    async () => {
+      // Trigger cron to update and issue receipts based on deals
+      const callDealerCronRes = await fetch(`https://staging.dealer.web3.storage/cron`)
+      t.true(callDealerCronRes.ok)
+
+      return receiptStoreFilecoin.get(aggregateAcceptReceiptCid)
+    },
```

Comment:
this seems unrelated to the PR title... can you explain why change?

Reply:
Added 

> Tests improved so that when things are still being processed in queue when we trigger cron, we don't await on outcome of cron run that ended up being NOP because queue is still processing.

Not documented before as I was trying this out when PR made...